addons/advanced_fatigue/stringtable.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="ACE">
+    <Package name="Advanced_Fatigue">
+        <Key ID="STR_ACE_Advanced_Fatigue_PerformanceFactor">
+            <English>Performance Factor</English>
+            <German>Leistungsfaktor</German>
+        </Key>
+        <Key ID="STR_ACE_Advanced_Fatigue_PerformanceFactor_Description">
+            <English>Influences the overall performance of all players with no custom factor. Higher means better.</English>
+            <German>Beinflusst die Leistungsfähigkeit aller Spieler ohne eigenen Leistungsfaktor. Ein höherer Wert heißt eine bessere Leistung.</German>
+        </Key>
+        <Key ID="STR_ACE_Advanced_Fatigue_PerformanceFactor_EdenDescription">
+            <English>Influences the overall performance of this unit. Higher means better.</English>
+            <German>Beinflusst die Leistungsfähigkeit dieser Einheit. Ein höherer Wert heißt eine bessere Leistung.</German>
+        </Key>
+        <Key ID="STR_ACE_Advanced_Fatigue_RecoveryFactor">
+            <English>Recovery Factor</English>
+            <German>Erholungsfaktor</German>
+        </Key>
+        <Key ID="STR_ACE_Advanced_Fatigue_RecoveryFactor_Description">
+            <English>Changes how long a player has to rest to regain stamina. Higher means faster.</English>
```

Comment:
Higher means *shorter* (previous sentence asks for "long", to which the antonym is "short")

Actually, I'd suggest a wording that intuitively conveys "higher is better": 
`How fast the player recovers when resting. Higher is faster.`
With that, you could maybe even do without the `Higher is faster` part, because the first sentence kind of naturally leads to that conclusion. 

/2c